app/containers/Vaults/migrationWhitelist.json

```diff
@@ -94,5 +94,21 @@
     "vaultTo": "0xa9fE4601811213c340e850ea305481afF02f5b28",
     "apyTooltip": "Please migrate funds to our new vault to continue earning yield.",
     "migrationMessage": "This vault is no longer active. Withdraw funds or migrate your balance to the new v2 WBTC yVault to continue earning yield."
+  },
+  {
+    "symbol": "crvBUSD",
+    "token": "0x3B3Ac5386837Dc563660FB6a0937DFAa5924333B",
+    "vaultFrom": "0x2994529C0652D127b7842094103715ec5299bBed",
```

Comment:
@dudesahn can you double check these are the correct addresses?

Reply:
@x48-crypto vault from was confirmed by @fameal. Maybe you can confirm tokens too @fameal

Reply:
Confirmed, addresses are good